src/app/page.tsx

```diff
@@ -1,8 +1,15 @@
 import Image from "next/image";
+import { JSX } from "react";
 
-export default function Home() {
+export function sum(a: number, b: number) {
+  return a + b;
+}
+
+export default function Home(): JSX.Element {
   const num = 1;
-  const str = "Hello, world!";
+  const str = "Hello, ReviewDog!";
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'str' is assigned a value but never used.
<!-- __reviewdog__:ChA1NzIyYjFhYjI0NDA0ZGU1EgZlc2xpbnQ= -->